extensions/helix/package.build.ts

```diff
@@ -0,0 +1,217 @@
+// Save to package.json
+// ============================================================================
+
+import { Builder, generateIgnoredKeybinds } from "../../meta";
+import * as fs from "fs/promises";
+import { SelectionBehavior } from "../../src/api";
+
+const version = "0.1.0",
+      preRelease = 1,
+      preReleaseVersion = version.replace(/\d+$/, "$&" + preRelease.toString().padStart(3, "0"));
+
+export const pkg = (modules: Builder.ParsedModule[]) => ({
+
+  // Common package.json properties.
+  // ==========================================================================
+
+  name: "dance-helix-keybinds",
+  description: "Helix keybindings for dance",
+  version,
+  license: "ISC",
```

Comment:
```suggestion
  license: "ISC",

  extensionDependencies: ["gregoire.dance"],
```

(IIRC there are constants somewhere with the name of the publisher / the name of the extension, but I forgot where)

Reply:
Found it ! `src/utils/constants.ts`, which does work yeah.